apps-rendering/src/date.test.ts

```diff
@@ -91,6 +92,20 @@ describe('formatLocal', () => {
 	});
 });
 
+describe('formatLocalTimeDateTz', () => {
+	test('returns correct local time zone (local timezone is Europe/London set in TZ global variable)', () => {
+		expect(formatLocalTimeDateTz(new Date('2020-03-11T17:25:00'))).toBe(
+			'5:25pm 11 Mar 2020 GMT',
+		);
```

Comment:
Just a random thought, what causes this code to decide the local timezone is `GMT`? Could it make a different choice if it's running on a CI server based in a different timezone?

Reply:
The timezone is defined in the setup (https://github.com/guardian/dotcom-rendering/blob/main/apps-rendering/config/jestglobalSetup.js#L5), so will always be consistent